solutions/devpsrint-michael-douglas-1/GitHubApp/Screens/List/RepositoryCellView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ListViewCell.swift
+//  GitHubApp
+//
+//  Created by Rogerio Cardoso Filho on 18/01/22.
+//
+
+import UIKit
+
+class RepositoryCellView : UITableViewCell {
+    
+    /* **************************************************************************************************
+    **
+    **  MARK: Variables Declaration
+    **
+    ****************************************************************************************************/
+    
+    var repositoryLabel : UILabel!
```

Comment:
Uma outra coisa, evite sempre utilizar Force Unwrap `!`, a menos que estas sejam propriedades criadas pelo próprio Interface Builder com @IBOutlet.

Caso contrário, busque sempre instanciar as propriedades via Propriedades Computadas e até mesmo com `lazy` vars.

Lembrando que a questão dos 2 pontos se aplica a tudo, nesse caso o correto é `repositoryLabel: UILable`.

Exemplo:
```suggestion
    lazy var repositoryLabel: UILabel = {
        let label = UILabel()
        label.textColor = UIColor.black
        label.font = UIFont.boldSystemFont(ofSize: 16)
        return label
    }()
```